RELEASE.txt

```diff
@@ -4,6 +4,12 @@ RELEASE=2.1.2
 
 # Release History
 
+## 2.2 Pre-release notes
+ - Backwards compatibility breaks
+   - A 'subtitle' is now required for all permuted tests.
```

Comment:
I guess this isn't true any longer, right?

Reply:
damnit jim.